modules/cnv-creating-interface-on-nodes.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-creating-interface-on-nodes_{context}"]
+= Creating an interface on nodes
+
+Create an interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster. The manifest details the desired configuration for the interface. 
+
+By default, the manifest will apply to all nodes in the cluster. To add the interface to specific nodes, add the `spec: nodeSelector:` parameter and the appropriate `<key>:<value>` for your node-selector.
+
+.Procedure
+
+. Create the `NodeNetworkConfigurationPolicy` manifest. The following example configures a linux bridge on two nodes:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: <br1-eth1-policy> <1>
+spec:
+  nodeSelector: <2>
+    kubernetes.io/hostname: <node01>,<node02> <3>
+  desiredState:
+    interfaces:
+      - name: br1
+        description: <Linux bridge with eth1 as a port>
```

Comment:
No need for `<>`. Just keep it as a plain string with a note that this is an optional description.